src/Model/BacktraceData.cs

```diff
@@ -146,8 +146,8 @@ public static BacktraceData Deserialize(string json)
         {
             var @object = BacktraceJObject.Parse(json);
 
-            var classfiers = @object["classifiers"]?
-                .Select(n => n.Value<string>()).ToArray() ?? null;
+            var classfiers = @object == null ? null : @object["classifiers"]
+                                 .Select(n => n.Value<string>()).ToArray() ?? null;

```

Comment:
`?? null`  - I think its not compatible with .NET 2.0.  I will prepare fix for this change. We don't need to use `??` here.